Lares/Assets/Player/Scripts/PlayerController.cs

```diff
@@ -58,16 +59,21 @@ private set
         [SerializeField] private float _maxRaycastHitDistance = 1.0f;
         [SerializeField] private float _maxInclineAngle = 60.0f;
 
+        
+
         void Start()
         {
             _rigidbody = GetComponent<Rigidbody>();
             _input = GetComponent<PlayerInput>();
 
-            _input.currentActionMap.FindAction("MovePlayer").performed += OnPlayerStartMoving;
-            _input.currentActionMap.FindAction("MovePlayer").canceled += OnPlayerStopMoving;
-            _input.currentActionMap.FindAction("Jump").performed += Jump;
-            _input.currentActionMap.FindAction("Sprint").performed += OnPlayerSprint;
-            _input.currentActionMap.FindAction("Sprint").canceled += OnPlayerSprint;
+            InputActionMap actionMap = _input.actions.FindActionMap("BaseControls");
+
+            actionMap.FindAction("MovePlayer").performed += OnPlayerStartMoving;
+            actionMap.FindAction("MovePlayer").canceled += OnPlayerStopMoving;
+            actionMap.FindAction("Jump").performed += Jump;
```

Comment:
Somehow missed this before but:
```suggestion
            actionMap.FindAction("Jump").performed += OnJump;
```

Will need the function changed too.